README.md

```diff
@@ -25,23 +25,23 @@ import (
 )
 
 func main() {
-    var (
-        err error
-        src io.Reader
-    )
-
-    if src, err = os.Open("score.yaml"); err != nil {
+    src, err := os.Open("score.yaml")
+    if err != nil {
         panic(err)
     }
-	defer src.Close()
+    defer src.Close()
     
     var srcMap map[string]interface{}
-    if err = loader.ParseYAML(&srcMap, src); err != nil {
+    if err := loader.ParseYAML(&srcMap, src); err != nil {
         panic(err)
     }
     
     var spec score.WorkloadSpec
```

Comment:
We should also recommend a call to score-go/schema Validate() before calling map spec.

Reply:
Added.